frontend/src/components/LoginPage.tsx

```diff
@@ -71,6 +82,10 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
     dispatch(setUsername(username));
     dispatch(connectWebSocket(username));
   },
+  changeView: (username: string) => {
```

Comment:
Consider using react-router for this logic (we could use the `/` route for the old game logic and `/stats` to render stats)

Reply:
Never mind, this seems good enough :stuck_out_tongue:

Reply:
Let's avoid configuring routing :P